conf/evolutions/default/85.sql

```diff
@@ -0,0 +1,12 @@
+# --- !Ups
+
+INSERT INTO `vitals` (`id`, `name`, `data_type`, `unit_of_measurement`)
+  VALUES ('11', 'weeksPregnant', 'float', 'weeks');
```

Comment:
also -- data_type can probably just be int. I'm not sure if anyone would need to record 9.5 weeks pregnant or not. Thoughts?